config/ide-helper.php

```diff
@@ -50,6 +50,22 @@
 
     'write_model_magic_where' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Write Model relation count properties
+    |--------------------------------------------------------------------------
+    |
+    | Set to false to disable writing of relation count properties to model DocBlocks.
+    |
+    | If set to true, every "x to many" relation will generate an accompanying
+    | "$relation_count" property DocBlock comment.
+    |
+    | If set to false, no such DocBlock comments will be generated.
```

Comment:
This seems a bit redundant, i think the description is long and clear enough without this line.

Reply:
I second this, please remove it